dataactcore/models/stagingModels.py

```diff
@@ -1,15 +1,261 @@
+
+class AwardFinancialAssistance(Base):
+    """Model for the award_financial_assistance table."""
+    __tablename__ = "award_financial_assistance"
+
+    award_financial_assistance_id = Column(Integer, primary_key=True)
+    actiondate = Column(Text, nullable=False)
+    actiontype = Column(Text)
+    assistancetype = Column(Text, nullable=False)
+    awarddescription = Column(Text)
+    awardeeorrecipientlegalentityname = Column(Text, nullable=False)
+    awardeeorrecipientuniqueidentifier = Column(Text)
+    awardingagencycode = Column(Text)
+    awardingagencyname = Column(Text)
+    awardingofficecode = Column(Text)
+    awardingofficename = Column(Text)
+    awardingsubtieragencycode = Column(Text, nullable=False)
+    awardingsubtieragencyname = Column(Text)
+    awardmodificationamendmentnumber = Column(Text)
+    businessfundsindicator = Column(Text, nullable=False)
+    businesstypes = Column(Text, nullable=False)
+    cfda_number = Column(Text, nullable=False)
+    cfda_title = Column(Text)
+    correctionlatedeleteindicator = Column(Text)
+    facevalueloanguarantee = Column(Numeric)
+    fain = Column(Text)
+    federalactionobligation = Column(Numeric)
+    fiscalyearandquartercorrection = Column(Text)
+    fundingagencycode = Column(Text)
+    fundingagencyname = Column(Text)
+    fundingagencyofficename = Column(Text)
+    fundingofficecode = Column(Text)
+    fundingsubtieragencycode = Column(Text)
+    fundingsubtieragencyname = Column(Text)
+    legalentityaddressline1 = Column(Text)
+    legalentityaddressline2 = Column(Text)
+    legalentityaddressline3 = Column(Text)
+    legalentitycitycode = Column(Text)
+    legalentitycityname = Column(Text)
+    legalentitycongressionaldistrict = Column(Text)
+    legalentitycountrycode = Column(Text)
+    legalentitycountycode = Column(Text)
+    legalentitycountyname = Column(Text)
+    legalentityforeigncityname = Column(Text)
+    legalentityforeignpostalcode = Column(Text)
+    legalentityforeignprovincename = Column(Text)
+    legalentitystatecode = Column(Text)
+    legalentitystatename = Column(Text)
+    legalentityzip5 = Column(Text)
+    legalentityziplast4 = Column(Text)
+    nonfederalfundingamount = Column(Numeric)
+    originalloansubsidycost = Column(Numeric)
+    periodofperformancecurrentenddate = Column(Text)
+    periodofperformancestartdate = Column(Text)
+    primaryplaceofperformancecityname = Column(Text)
+    primaryplaceofperformancecode = Column(Text)
+    primaryplaceofperformancecongressionaldistrict = Column(Text)
+    primaryplaceofperformancecountrycode = Column(Text)
+    primaryplaceofperformancecountyname = Column(Text)
+    primaryplaceofperformanceforeignlocationdescription = Column(Text)
+    primaryplaceofperformancestatename = Column(Text)
+    primaryplaceofperformancezipplus4 = Column(Text)
+    recordtype = Column(Integer)
+    sai_number = Column(Text)
+    totalfundingamount = Column(Numeric)
+    uri = Column(Text)
+
```

Comment:
This and file C (award financial) could use an index on FAIN and URI.  File C will also eventually need one on PIID and ParentAwardId.